@here/harp-text-canvas/lib/TextCanvas.ts

```diff
@@ -14,6 +14,7 @@ import { SdfTextMaterial } from "./rendering/TextMaterials";
 import { FontVariant, TextLayoutStyle, TextRenderStyle } from "./rendering/TextStyle";
 import { LineTypesetter } from "./typesetting/LineTypesetter";
 import { PathTypesetter, PathTypesettingParameters } from "./typesetting/PathTypesetter";
+import { SmoothPathTypesetter } from "./typesetting/SmoothPathTypesetter";
```

Comment:
I think this PR would be a nice opportunity to explore how the API could support user-defined typesetters, so we could take this code (and maybe the path typesetter too) outside of the module and keep it as part of the application logic, so maintainability of this module doesn't suffer...

But I can live with doing this later.